debian/patches/no-single-process.patch

```diff
@@ -251,3 +251,68 @@ Last-Update: 2024-08-02
 +
 +[Install]
 +WantedBy=cloud-init.target
+--- a/cloudinit/cmd/status.py
++++ b/cloudinit/cmd/status.py
+@@ -318,9 +318,8 @@
+     for service in [
+         "cloud-final.service",
+         "cloud-config.service",
+-        "cloud-init-network.service",
++        "cloud-init.service",
+         "cloud-init-local.service",
+-        "cloud-init-main.service",
+     ]:
+         try:
+             stdout = query_systemctl(
+--- a/cloudinit/config/cc_mounts.py
++++ b/cloudinit/config/cc_mounts.py
+@@ -525,7 +525,7 @@
+     # fs_spec, fs_file, fs_vfstype, fs_mntops, fs-freq, fs_passno
+     uses_systemd = cloud.distro.uses_systemd()
+     default_mount_options = (
+-        "defaults,nofail,x-systemd.after=cloud-init-network.service,_netdev"
++        "defaults,nofail,x-systemd.after=cloud-init.service,_netdev"
+         if uses_systemd
+         else "defaults,nobootwait"
+     )
+--- a/cloudinit/config/schemas/schema-cloud-config-v1.json
++++ b/cloudinit/config/schemas/schema-cloud-config-v1.json
+@@ -2022,12 +2022,12 @@
+         },
+         "mount_default_fields": {
+           "type": "array",
+-          "description": "Default mount configuration for any mount entry with less than 6 options provided. When specified, 6 items are required and represent ``/etc/fstab`` entries. Default: ``defaults,nofail,x-systemd.after=cloud-init-network.service,_netdev``.",
++          "description": "Default mount configuration for any mount entry with less than 6 options provided. When specified, 6 items are required and represent ``/etc/fstab`` entries. Default: ``defaults,nofail,x-systemd.after=cloud-init.service,_netdev``.",
+           "default": [
+             null,
+             null,
+             "auto",
+-            "defaults,nofail,x-systemd.after=cloud-init-network.service",
++            "defaults,nofail,x-systemd.after=cloud-init.service",
+             "0",
+             "2"
+           ],
+--- a/systemd/cloud-config.target
++++ b/systemd/cloud-config.target
+@@ -14,5 +14,5 @@
+ 
+ [Unit]
+ Description=Cloud-config availability
+-Wants=cloud-init-local.service cloud-init-network.service
+-After=cloud-init-local.service cloud-init-network.service
++Wants=cloud-init-local.service cloud-init.service
++After=cloud-init-local.service cloud-init.service
```

Comment:
Thanks for catching this. I just checked with the patch applied and I don't see any other references that should require changing, just comments, docs, and tests

Reply:
Don't we need a changelog entry for this commit?

Reply:
Good thought, but not here because we just introduced this unreleased quilt patch in the latest changelog commit. `  * d/p/no-nocloud-network.patch: Remove nocloud network feature`. So, no need to ammend that new entry.